client/jetpack-connect/authorize.js

```diff
@@ -255,15 +254,15 @@ export class JetpackAuthorize extends Component {
 	}
 
 	shouldRedirectJetpackStart( props = this.props ) {
-		const { partnerId } = props.authQuery;
+		const { partnerSlug } = props.authQuery;
 		const partnerRedirectFlag = config.isEnabled(
 			'jetpack/connect-redirect-pressable-credential-approval'
 		);
 
 		// If the redirect flag is set, then we conditionally redirect the Pressable client to
 		// a credential approval screen. Otherwise, we need to redirect all other partners back
 		// to wp-admin.
-		return partnerRedirectFlag ? partnerId && PRESSABLE_PARTNER_ID !== partnerId : partnerId;
+		return partnerRedirectFlag ? partnerSlug && 'pressable' !== partnerSlug : partnerSlug;
```

Comment:
Previously, this would only match `49640` id. Now it will match either 49615 or 49640 pressable ID. Is that OK?

Reply:
The desired behavior here is that we want to redirect users that are getting a plan provisioned by Pressable to another page to approve credential sharing. `49615` is Pressable's testing key. So, I don't imagine how it'd be an issue.